AnkiDroid/src/main/java/com/ichi2/libanki/Consts.kt

```diff
@@ -93,7 +93,7 @@ object Consts {
     // The labels defined in consts.py are in AnkiDroid's resources files.
     const val DEFAULT_DECK_ID: Long = 1
 
-    val FIELD_SEPARATOR = Character.toString('\u001f')
+    val FIELD_SEPARATOR: String = Character.toString('\u001f')
```

Comment:
might as well fix all the issues on the line while you're here

```suggestion
    const val FIELD_SEPARATOR: String = '\u001f'.toString()
```